.github/minty.yaml

```diff
@@ -4,7 +4,8 @@ rule:
   if: |-
     assertion.iss == issuers.github &&
     assertion.repository_owner_id == '93787867' &&
-    assertion.repository_id == '576289489'
+    assertion.repository_id == '576289489' ||
+    assertion.iss == issuers.google && assertion.sub == '112892967544648233217'
```

Comment:
I think we may need to add some parens for grouping?

```cel
(
  assertion.iss == issuers.github &&
  assertion.repository_owner_id == '93787867' &&
  assertion.repository_id == '576289489'
) || (
  assertion.iss == issuers.google &&
  assertion.sub == '112892967544648233217'
)
```

Reply:
Done